scanner/src/main/kotlin/scanners/fossid/FossIdConfig.kt

```diff
@@ -206,4 +206,9 @@ internal data class FossIdConfig(
 
         return FossIdNamingProvider(namingProjectPattern, namingScanPattern, namingConventionVariables)
     }
+
+    /**
+     * Create a [FossIdUrlProvider] helper object based on the configuration stored in this object.
+     */
+    fun createUrlProvider(): FossIdUrlProvider = FossIdUrlProvider.create(addAuthenticationToUrl)
```

Comment:
Nit: Maybe drop the explicit return type to avoid the `FossIdUrlProvider = FossIdUrlProvider` duplication?